### ISSUE TITLE
Biojuice effect adds to core corruption stat instead of potion effects.

### ISSUE THREAD

**Issue by Nora168** [Innoxia/liliths-throne-public#944]
The effect says that it adds 50 corruption to 'potion effects' but instead adds 50 core corruption which is a little expensive to remove with potions... even with the halved cost from the transformer fetish it still takes 301 essences to remove 50 core corruption.

**Comment by BluefoxKitsune** [comment]
If you're interested in actually removing that without resorting to the cheat menu, I recommend making a few milk slaves and farming them for money. Fill up a dairy room and it's an easy 100k a day, with which you can buy essences.

**Comment by Nora168** [comment]
Yeah thanks for the idea but I'm already doing that. My 6 slaves that I use for my milking room make 215k a day and the rest of my slaves make 34k a day after expenses. I'm not even overworking anyone and every room has every upgrade besides the arcane instruments.  _Btw a Double room with room service, double bed, and obedience trainer give +0.35 affection and +0.10 obedience a day... not a lot but keeps everything from going down. as for getting it up. Give everyone a slight desire for masochist and exhibitionist... then inspecting, spanking and molesting give positive boosts instead as long as they're attracted to you._ 

Plus to get around using the biojuice I just use either the slime queen or the first slime relating to that quest to transform myself free of all costs.

So Money isn't really the issue. I just can't decide between being a Fox-morph or a slime... so I end up going back and forth a lot. Regardless of how I transform it still feels like I have to go out of my way to do it. Although it wouldn't be an issue at all if I just picked one or the other lol. Honestly it'd be nice if there was a spell or single item I could carry around to transform myself between the two.

I tried making a mod item with the biojuice effect... and it worked... it just also gave the core curruption...
